recipes/igblast/meta.yaml

```diff
@@ -1,36 +1,48 @@
+{% set version = "1.15.0" %}
+
 package:
   name: igblast
-  version: "1.10.0"
+  version: "{{ version }}"
 
 about:
   home: http://www.ncbi.nlm.nih.gov/projects/igblast/
-  license: Public Domain
-  summary: A tool for analyzing immunoglobulin (IG) and T cell receptor (TR) sequences.
+  license: Public Domain and others
```

Comment:
Surely " and others" is not ideal yet, but having the additional license files and it not just say "Public Domain" is a good improvement nonetheless! Thanks for that.

Reply:
No, I’m aware of that, but at the moment, the values used as `license` are all quite inconsistent within Bioconda anyway. It would be great if Bioconda would start to require SPDX license expressions here, though.

Reply:
Yep, I've been a proponent of SPDX license expressions for a good while now. Just very recently we finally have linter recommendations for them over at conda-forge, which is a good start!
Tooling for license information handling in Bioconda (and `conda` itself, actually) is on my list for this year ;) (if/when I can hopefully work more on Bioconda, that is).

Reply:
Oh, that would be awesome if you coul pull this off!